.github/workflows/update-aaguids.yml

```
@@ -0,0 +1,36 @@
+name: Update AAGUIDs
+
+on:
+  schedule:
+    - cron: "0 0 * * 0" # Runs every Sunday at midnight
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  update-aaguids:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install jq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
+
+      - name: Fetch JSON data
+        run: |
+          curl -o data.json https://raw.githubusercontent.com/passkeydeveloper/passkey-authenticator-aaguids/refs/heads/main/combined_aaguid.json
+
+      - name: Process JSON data
+        run: |
+          mkdir -p backend/resources
+          jq 'map_values(del(.icon_light, .icon_dark))' data.json > backend/resources/aaguids.json
```

Use `-c` to minify the output. Since the JSON will be embedded in the binary, we don't care if it's pretty-printed

```suggestion
          jq -c 'map_values(del(.icon_light, .icon_dark))' data.json > backend/resources/aaguids.json
```

---

I mean sure inside the binary but the file still exists in the code base, so in the event we ever need to reference it i think leaving it un-minified is fine.

---

But you are loading additional, unnecessary bytes into the executable, which are kept inside memory pages. So this does have an impact at runtime too.